pkgs/tools/typesetting/tex/texlive/fixHashes.awk

```diff
@@ -10,7 +10,7 @@ BEGIN {
     print "No match for \""$0"\"" > "/dev/stderr"
     exit 1
   }
-  cmd="nix-hash --type sha1 --base32 "$0
+  cmd="nix-hash --type sha256 --base32 "$0
```

Comment:
If we are switching hashes, we might as well switch from base32 to sri. However, I don't see support for it in nix-hash. Maybe we could use `nix hash file` instead

Reply:
The nix command set is unstable, right? Maybe not a good idea to use unstable stuff in nixpkgs?

Reply:
Yes, but this is not used by users, only by maintainers. There are many sri hashes in nixpkgs already, and the big chunk of those must have come from the use of nix command.

Reply:
so whats the verdict here?

Reply:
I think it's fine to use `nix hash` then, with the caveat that the script might not work for automation and/or maintainers using the stable version of nix.

Reply:
Let's merge this as is, feel free to create another PR if you need to change hashes to sri.

Reply:
@veprbl @Mindavi would a nix expression good for generating the fixed hashes (something to call `nix-build` on)? Or are there reasons for keeping it as just a script?

Reply:
I'm not sure what the best way to handle this is.

Reply:
> would a nix expression good for generating the fixed hashes (something to call `nix-build` on)?

That would require "recursive nix".

> Or are there reasons for keeping it as just a script?

Because the script is working fine.